<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QWidget" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="m_layoutMain">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_container" native="true">
     <layout class="QVBoxLayout" name="m_layoutContainer">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="LabelWithStatus" name="m_lblStatus" native="true"/>
        </item>
        <item>
         <widget class="PlainToolButton" name="m_btnPlayPause">
          <property name="toolTip">
           <string>Play/pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlainToolButton" name="m_btnStop">
          <property name="toolTip">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_spinSpeed">
          <property name="toolTip">
           <string>Speed</string>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string notr="true"> %</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="m_slidProgress">
          <property name="toolTip">
           <string>Progress</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_lblTime">
          <property name="toolTip">
           <string>Duration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlainToolButton" name="m_btnVolume">
          <property name="toolTip">
           <string>Mute/unmute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="m_slidVolume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Volume</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlainToolButton" name="m_btnDownload">
          <property name="toolTip">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlainToolButton" name="m_btnFullscreen">
          <property name="toolTip">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlainToolButton</class>
   <extends>QToolButton</extends>
   <header>plaintoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btnPlayPause</tabstop>
  <tabstop>m_btnStop</tabstop>
  <tabstop>m_spinSpeed</tabstop>
  <tabstop>m_slidProgress</tabstop>
  <tabstop>m_btnVolume</tabstop>
  <tabstop>m_slidVolume</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
